source/gui/settingsDialogs.py

```diff
@@ -35,6 +35,18 @@
 	updateCheck = None
 import inputCore
 
+#Choices for numbers combo box in Voice settings.
+
+DIGIT_CHOICES = [
+	# Translators: Choice in a combo box for speaking numbers by word.
+	_("Words"),

```

Comment:
I just took a quick look on your code, looks great. The only idea I'd like to share is to change the "words" option to "controlled by synthesizer", since it is up to the synthesizer what to do with numbers.